.mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
```

Comment:
Would it make sense to start using `pyproject.toml` right away, and add the mypy settings there? E.g. pylint can also store its configuration there.

Reply:
Flake8 can't sadly https://github.com/PyCQA/flake8/issues/234